susemanager-utils/susemanager-sls/src/modules/mgrnet.py

```diff
@@ -0,0 +1,92 @@
+"""
+Module for gathering DNS FQDNs
+"""
+
+import logging
+import re
+import time
+from multiprocessing.pool import ThreadPool
```

Comment:
Could you share why you opted for `multiprocessing.pool.ThreadPool`? What's the advantage over `multiprocessing.Pool` and the `threading` module?

Reply:
I think you are asking good question. I have no idea why `multiprocessing` was used originally for `network` module. Actually this part is more or less a copy of the logic from `network` module.
I'll check with `threading` and update the PR.